### PR TITLE
Add Editorconfig

### PR DESCRIPTION
**Pull request by denysdovhan**
See: http://editorconfig.org/

### CODE REVIEW

**Comment by denysdovhan** [comment]
Would you prefer to use this set of settings?

```
# http://editorconfig.org
root = true

[*]
indent_style = tab
indent_size = 4
end_of_line = crlf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```

**Comment by Falke-Design** [comment]
Because there was no request for this since 2018 anymore, I suggest to close this

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# EditorConfig
+# http://EditorConfig.org
```
Comment (Malvoz) [inline .editorconfig:2]:
```suggestion
# https://EditorConfig.org
```

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.js]
+indent_style = tab
+tab_width = 4
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
```
Comment (Malvoz) [inline .editorconfig:20]: What's the basis for these settings? They'd have to be kept in sync with the ESLint config right? (https://github.com/Leaflet/Leaflet/issues/8176 / https://github.com/Leaflet/Leaflet/issues/3870)

**Review by jonkoops — CHANGES_REQUESTED** [review]
I don't see the value of adding a config that can become out of sync with our existing linting tools (which are checked by CI). Checking this config is also in no way enforced by any tooling on CI at the moment.

**Comment by Malvoz** [comment]
Closing per https://github.com/Leaflet/Leaflet/pull/6125#pullrequestreview-951256183, https://github.com/Leaflet/Leaflet/pull/6125#issuecomment-968363321 and https://github.com/Leaflet/Leaflet/pull/2347#issuecomment-32362778.

**Pull request closed without merging**